ansible-scylla-node/tasks/common.yml

```diff
@@ -390,14 +390,37 @@
           {% if (_scylla_yaml_map.audit is defined and _scylla_yaml_map.audit == 'table') %}True{% else %}False{% endif %}
   run_once: true
 
+- name: Check if all nodes are up
+  block:
+    - name: Use Scylla API to see if the nodes are up
+      uri:
+        url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/failure_detector/endpoints/"
+        follow_redirects: none
+        method: GET
+      register: _failure_detector_result
+      until: _failure_detector_result.status == 200
+      retries: 3
+      delay: 1
+      ignore_errors: true
+      delegate_to: "{{ item }}"
+      loop: "{{ groups['scylla'] }}"
```

Comment:
Why running this serially and not concurrently (without loop)? Imagine that your cluster has 100s of nodes - this will run forever!

Reply:
Fixed